source/components/JsxParser.tsx

```diff
@@ -73,26 +74,26 @@ export default class JsxParser extends React.Component<TProps> {
 			return null
 		}
 
-		return parsed.map(this.#parseExpression).filter(Boolean)
+		return parsed.map(p => this.#parseExpression(p)).filter(Boolean)
 	}
 
-	#parseExpression = (expression: AcornJSX.Expression): any => {
+	#parseExpression = (expression: AcornJSX.Expression, scope?: Scope): any => {
```

Comment:
I don't feel particularly wonderful about adding `scope` everywhere, but i'm not sure of a better way to allow access to the function params in subsequent expressions.

Reply:
Agreed - it feels a little odd. At the same time, this entire thing is a recursive algorithm, parsing a tree - so allowing the parent context to pass along the current scope seems reasonable. It's essentially allowing the sap to run from the trunk of the tree down the branches, which is fine.